client/src/locales/gb.json

```
@@ -445,73 +445,623 @@
+    "selectFileTips": "Select CSV file to upload",
+    "selectFileDescription": "You can download our <template>template</template> or <sample>sample</sample>",
+    "selectFile": "Select File",
+    "parsingFailed": "Parsing failed",
+    "uploadSuccess": "Monitors created successfully!",
+    "validationFailed": "Validation failed",
+    "noFileSelected": "No file selected",
+    "fallbackPage": "Import a file to upload a list of servers in bulk"
+  },
+  "publicLink": "Public link",
+  "maskedPageSpeedKeyPlaceholder": "*************************************",
+  "pageSpeedApiKeyFieldTitle": "Google PageSpeed API key",
```

Biome (1.9.4)

[error] The key pageSpeedApiKeyFieldTitle was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)